_ping.php

```diff
@@ -38,15 +38,26 @@ function status_shutdown() {
 }
 // Check that all memcache instances are running on this server.
 if (isset($settings['memcache']['servers'])) {
```

Comment:
$fails variable needs to be initialized here ($fails = array(); or $fails = [];)

Otherwise execution of php unit test cases will result AssertionFailedError.

```[PHPUnit\Framework\AssertionFailedError]
  php - Notice: <em class="placeholder">Notice</em>: Undefined variable: fails in <em class="placeholder">main()</em> (line <em class="placeholder">58</em> of <em class="pla
  ceholder">/var/www/nordicspirits.dev.wunder.io/releases/1503982206/web/_ping.php</em>)```